.github/scripts/install-libff.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -eux -o pipefail
+
+## The following script builds and installs libff to ~/.local/lib
+
+INSTALL_VERSION=v0.2.1
+
+if [[ "$(uname -s)" =~ ^MSYS_NT.* ]]; then
+    echo "This script is only meant to run on Windows under MSYS2"
+    exit 1
+fi
+
+if [ -f "$HOME/.local/lib/libff.a" ]; then
+  echo "libff exists, exiting..."
+  exit 0
+fi
+
+if [ -d libff ]; then
+  echo "$(pwd)/libff" already exists! Using it instead of re-cloning the repo.
+else
```

Comment:
Are we expecting these lines to ever be executed in our ci env?

Reply:
I don't think so, as the CI runs on a clean slate each time, but it's handy if you're (re)running the script locally. Do you want me to drop it so it's a bit simpler?